pkg/server/singleprocess/service_runner.go

```diff
@@ -874,3 +880,50 @@ func (s *Service) runnerVerifyToken(
 
 	return nil
 }
+
+func operationString(job *pb.Job) string {
```

Comment:
we seem to do this in more than one place in Waypoint, so I'm open to suggestions to where we could place this and use it. Maybe [/pkg/server/ptypes/job.go](https://github.com/hashicorp/waypoint/blob/bc8959caca4047ab142dff101ad2d32a43042431/pkg/server/ptypes/job.go)?

Reply:
The CLI client needs something like this too, so maybe `pkg/server/...` isn't the right place. Maybe it could go some where in `internal/...`? 🤔